core/station/impl/src/services/system.rs

```diff
@@ -468,6 +477,22 @@ mod init_canister_sync_handlers {
         );
     }
 
+    pub fn add_initial_assets() {
+        let initial_assets: Vec<Asset> = vec![Asset {
+            blockchain: Blockchain::InternetComputer,
+            standards: BTreeSet::from([BlockchainStandard::Native]),
+            symbol: "ICP".to_string(),
+            name: "Internet Computer".to_string(),
+            metadata: Metadata::default(),
```

Comment:
should the mainnet `ledger` and `index` canister id be here?

Reply:
In this PR, I don't think it's necessary since we'll get those global constants refactored anyways when accounts will contain AccountAsset's.